contracts/hardhat.config.ts

```diff
@@ -131,7 +131,6 @@ const config: HardhatUserConfig = {
     gethDev: {
       url: `http://localhost:8545`,
       accounts,
-      gasPrice: 0,
```

Comment:
Sending transactions hangs with `gasPrice:0` when using geth, probably because geth is unaware of this option.

I believe this was an early hack that we've outgrown anyway. Transactions need to pay gas.